app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -675,11 +758,11 @@ class SettingsFragment : Fragment(R.layout.activity_settings_screen) {
                     //creating json object
                     val jsonObject = JSONObject(stringResponse)
                     val version = jsonObject.getInt(Constants.JSON_VERSION)
-                    if (DEBUG) Log.d(LOG_TAG, "$FILETAG client onResponse for refresh blocklist files-  $version")
+                    if (DEBUG) Log.d(LOG_TAG_DOWNLOAD, "$FILETAG client onResponse for refresh blocklist files-  $version")
```

Comment:
get rid of filetag, or use it as log-tag.

Reply:
Removed `filetag`